### PR TITLE
date-picker added

### PR DESCRIPTION
**Pull request by uglygus**
Adds a new plugin: date-picker

### CODE REVIEW

**Comment by uglygus** [comment]
The build fails from a ruby version mismatch. Its building with 2.0 and rubocop needs >=2.1. I'm not sure if there's anything I can do about it.

...
Using /Users/travis/.rvm/gems/ruby-2.0.0-p648
$ ruby --version
ruby 2.0.0p648 (2015-12-16 revision 53162) [x86_64-darwin15.5.0]
....
Fetching: rubocop-0.51.0.gem (100%)
ERROR: Error installing rubocop:
	rubocop requires Ruby version >= 2.1.0.
...

File: Time/date-picker.1m.sh [hunk 1]
```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Display todays date and time in various formats including ISO8601 and allows copying to clipboard.
+
+# Comment out the dates you don't need.
+
+# <bitbar.title>Date Picker/bitbar.title>
```
Comment (tresni) [inline Time/date-picker.1m.sh]: Missing the closing < on bitbar.title

File: Time/date-picker.1m.sh [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Cooper Battersby/bitbar.author>
```
Comment (tresni) [inline Time/date-picker.1m.sh]: Missing the closing < on bitbar.author

File: Time/date-picker.1m.sh [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+# <bitbar.author.github>uglygus</bitbar.author.github>
+# <bitbar.desc>Display todays date in various forms including iso8601 and copies to the clipboard.</bitbar.desc>
+# <bitbar.dependencies></bitbar.dependencies>
+# <bitbar.image>https://imgur.com/a/GTnzV</bitbar.image>
```
Comment (tresni) [inline Time/date-picker.1m.sh]: This should be the direct link to the image:  https://i.imgur.com/GVSUqFX.png

**Pull request merged**